vars/copyLogsFromSctRunner.groovy

```diff
@@ -12,6 +12,10 @@ def call(String sct_latest_dir) {
             eval \$(ssh-agent -k)
         }
         trap clean_ssh_agent EXIT
+
+        # still need to sync scylla-test, until replacing keys in jenkins
```

Comment:
TODO?

Reply:
on the followup, we'll be able to clear all of those..
(we'll search is base on the key names we need to clear out)